templates/d3js/index.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <title>ContainerLab Topology '{{ .Name }}'</title>
+    <style>
+      html, body{
+        width: 100%;
+        height: 100%;
+      }
+      #graph {
+        width: 80%;
+        height: 100%;
+      }
+      #content {
+        width: 80%;
+        height: 50%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="content" style="margin:0 auto;">
+      <h1>ContainerLab Topology '{{ .Name }}'</h1>
+      <div>
+        <button id='saveButton'>Export topology '{{ .Name }}' as PNG</button>
```

Comment:
can you add `class="button"` here to make this button inherit bootstrap styles?

Reply:
done